PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/test/utility/AccessPrivate.java

```diff
@@ -21,12 +21,6 @@
 public final class AccessPrivate {
 
   /**
-   * Don't let anyone instantiate this class.
```

Comment:
We should just exclude these annotation files from the coverage report (and all analysis tools), because they aren't part of our project.

Keep the constructor, it provides safety.